packages/cli/config/index.ts

```diff
@@ -618,6 +618,27 @@ const config = convict({
 		},
 	},
 
+	versionNotifications: {
+		enabled: {
+			doc: 'Whether feature to request new version notifications and security updates is enabled.',
```

Comment:
I think the security updates part is now misleading 🤔 

```suggestion
			doc: 'Whether feature is enabled to request notifications about new versions and security updates.',
```